Buildings/ThermalZones/Detailed/UsersGuide.mo

```diff
@@ -1017,6 +1017,45 @@ Proc. of the 2014 ASHRAE/IBPSA-USA Building Simulation Conference, Atlanta, GA,
 </p>
 </html>"));
   end CFD;
+
+  class ISAT
+    "Room model with air heat and mass balance computed using in situ adaptive tabulation"
+    extends Modelica.Icons.Information;
+  annotation (preferredView="info",
+  Documentation(info="<html>
+<p>The model <a href=\"modelica://Buildings.ThermalZones.Detailed.ISAT\">Buildings.ThermalZones.Detailed.ISAT</a> is a room model in which the room air heat and mass balance is computed using the In Situ Adaptive Tabulation (ISAT). </p>
+<p>The model is identical with <a href=\"modelica://Buildings.ThermalZones.Detailed.CFD\">Buildings.ThermalZones.Detailed.CFD</a>, except for the following points: </p>
+<ul>
+<li>The heat and mass balance of the air is computed using ISAT. </li>
+<li>The ISAT is a reduced-order self-learning storage-and-retrieve algorithm for fast prediction of indoor airflow and thermal environment.</li>
+<li>The ISAT is linked with FFD program, which means ISAT calls FFD and returns results from FFD simulation when ISAT can not retrive it due to lack of data in ISAT database.</li>
+<li>Since ISAT is a data-driven model, more inputs and outputs will cause signifiantly increasing computing effort. Therefore, in our current settings, assumed fixed values are assigned for the heat flow rates from ports and surfaces. </li>
+<li>In other words, the inputs of the ISAT room model are identical with the CFD room model, but for outputs of the ISAT room model, currently it only supports to return outputs including temp_roo, temp_occ, vel_occ, temp_sen, vel_sen and temp_rack.</li>
+<li>If the users would like to return heat flow rates for ports and surfaces, they should revise the source codes in cosimulation.c and utility_isat.c in <span style=\"font-family: Courier New;\">Buildings\\Resources\\src\\ISAT\\</span> by outputing target values and assign them to modelica-shared variables.</li>
+</ul>
+<p>A description of the model assumptions and the implemention and validation of this room model can be found in <a href=\"https://www.tandfonline.com/doi/full/10.1080/19401493.2017.1288761\">Tian et al. (2018)</a> and in <a href=\"https://www.researchgate.net/profile/Wangda_Zuo/publication/333797408_Optimization_of_Workload_Distribution_of_Data_Centers_Based_on_a_Self-Learning_In_Situ_Adaptive_Tabulation_Model/links/5d0467bf299bf12e7be02981/Optimization-of-Workload-Distribution-of-Data-Centers-Based-on-a-Self-Learning-In-Situ-Adaptive-Tabulation-Model.pdf\">Han et al. (2019)</a>. </p>
+<p>Before running examples of the ISAT model, a fortran compiler needs to be installed. Please refer to <a href=\"https://www.colorado.edu/lab/sbs/data-center-package\">ISAT project website</a> for more detailed tutorials.</p>
```

Comment:
Without installing the compiler, cannot user run the example models? Thought the ISAT library has already been compiled and user can run the model if no need to recompile.

Reply:
@JayHuLBL you are correct, the ISAT libraries have already been compiled and a Fortran compiler is no longer needed to run the models. This will be corrected in the next pull request.